api/infrastructure/parallelcluster-api.yaml

```diff
@@ -501,6 +501,37 @@ Resources:
                   aws:RequestedRegion:
                     - !Ref Region
             Sid: EC2Write
+          - Action:
+              - autoscaling:DescribeAutoScalingGroups
+              - autoscaling:DescribeScalingActivities
+              - autoscaling:DescribeAutoScalingInstances
+              - autoscaling:DescribeScheduledActions
+            Resource: '*'
+            Effect: Allow
+            Condition: !If
+              - IsMultiRegion
+              - !Ref AWS::NoValue
+              - StringEquals:
+                  aws:RequestedRegion:
+                    - !Ref Region
```

Comment:
if we add an arn in the Resource we can get rid of this and use the region or * in the arn

Reply:
done

Reply:
minor: to shorten the size of the policy can we use resource here as well with something like: `!Sub arn:${AWS::Partition}:autoscaling:${Region}:${AWS::AccountId}:*`>

Reply:
done